roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -149,19 +149,42 @@
     with_items:
       - {src: "{{ he_local_vm_dir }}/vm.conf", dest: /var/run/ovirt-hosted-engine-ha}
       - {src: "{{ he_local_vm_dir }}/hosted-engine.conf", dest: /etc/ovirt-hosted-engine/}
+  - name: Verify fapolicyd.rules file
+    stat:
+      path: /etc/fapolicyd/fapolicyd.rules
+    register: fapolicy_rules
+  - name: Add rule to fapolicyd.rules file
+    blockinfile:
+      path: /etc/fapolicyd/fapolicyd.rules
+      block: |
+        # temporary entry added by hosted-engine-setup for running commands as vdsm user
+        allow perm=any uid=36 : dir=/var/tmp/
```

Comment:
Instead of doing this hack, can't we just use different directory where this is allowed?

Reply:
Ansible files located in /var/tmp, not sure if we can and if it's a good idea to change it.
fapolicy added something similar for root (see https://bugzilla.redhat.com/show_bug.cgi?id=1746464).

Reply:
Please take a look into new feature in fapolicy [rules.d ](https://github.com/linux-application-whitelisting/fapolicyd/issues/117) that is going to be introduced in upcoming release fapolicy-1.1  (in RHEL 8.6).
 It's not there yet but I got information from its core developer that the feature will land in RHEL 8.6 nightly builds today/tomorrow.

An example (not yet verified) of the usage is in the PR I have prepared for Ovirt Engine's Wilddfly runtime tmp directory [here](https://github.com/linux-application-whitelisting/fapolicyd/issues/117)

Reply:
fapolicyd blocks ansible commands (located in /var/tmp) execution as non-root user.
For more details see https://bugzilla.redhat.com/1903549